mani_skill/envs/tasks/tabletop/test_stack_pyramid.py

```diff
@@ -0,0 +1,69 @@
+
```

Comment:
we don't usually have per env specific tests. this should be removed

Reply:
Gotcha. As a side question, I'm curious if there's a reason environments generally don't have per-environment tests?

My rationale for unit testing environment implementation was that it could help ensure the environment works correctly with current and future code changes. However, among different RL environments / framework, I've only come across one instance of environment-specific testing in Gymnasium, specifically in this file:
- https://github.com/Farama-Foundation/Gymnasium/blob/main/tests/envs/test_env_implementation.py